### PR TITLE
SQL Import: Poll for import status after kick off & in separate command

### PR DESCRIPTION
**Pull request by jblz**
## Description

Uses the job status from GraphQL to inform the user about the status of the import as it works through the system.

It leverages both the jobs system and the import_status site meta to relay the progress of the current import and to report failures to the user.

## Steps to Test

1. Check out PR.
1. Run `npm run build`
1. Run an import e.g. `./dist/bin/vip-import-sql.js @1000.production --search-replace="from.example.com,to.example.com" ~/Downloads/dbdump.sql`
    * Confirm job status polls appropriately
    * Confirm a successful import is reported correctly
1. Repeat the above, and exit the CLI prior to job completion (or proceed in a separate terminal session)
1. Run `./dist/bin/vip-import-sql-status.js @1000.production`
    * Confirm job status polls appropriately as above
1. Attempt to import various files that you know will fail server-side. (e.g. manually alter a db dump file to insert a row with a duplicate unique key)
    * Confirm the error is provided to you and is actionable.

### CODE REVIEW

File: src/bin/vip-import-sql.js [hunk 1]
```diff
@@ -45,22 +73,20 @@ const debug = debugLib( 'vip:vip-import-sql' );
 command( {
 	appContext: true,
 	appQuery,
-	requiredArgs: 1, // TODO print proper usage example
 	envContext: true,
+	requiredArgs: 1,
 	module: 'import-sql',
-	requireConfirm: 'Are you sure you want to import the contents of the provided SQL file?',
```
Comment (jblz) [inline src/bin/vip-import-sql.js:101]: reinstate this!

File: src/lib/cli/command.js [hunk 2]
```diff
@@ -329,6 +329,18 @@ args.argv = async function( argv, cb ): Promise<any> {
 		}
 	}
 
+	/*
+	 * All execution context has been calculated at this point
+	 * Use trackEventWithContext in place of trackEvent from here on out so it's included in event props
+	 */
+	const trackEventWithContext = async ( eventName, eventProps = {} ) => {
```
Comment (jblz) [inline src/lib/cli/command.js]: Should we pull this change into its own PR?

**Comment by elazzabi** [comment]
I added a top level status command to this part of https://github.com/Automattic/vip/pull/614/commits/1d4ee127dd1918aefabc5a12f7fc4634768ffed8, so a user can run:

```
vip import sql status @test.env
``` 

I'll do an end to end test soon

**Review by elazzabi — APPROVED** [review]
End-to-end testing looks good from my end. I shared a video on Slack.

As part of another PR, we can work on making the spinning step a little quicker.
